Source/WebGPU/WebGPU/APIConversions.h

```diff
@@ -59,142 +59,142 @@ namespace WebGPU {
 
 // FIXME: It would be cool if we didn't have to list all these overloads, but instead could do something like bridge_cast() in WTF.
 
-inline Adapter& fromAPI(WGPUAdapter adapter)
+inline Ref<Adapter> fromAPI(WGPUAdapter adapter)
```

Comment:
We don't typically do this. Instead, we instantiate Ref / RefPtr object in stack where the result of the conversion is used.

Reply:
Good to know, thank you. How does that differ from the following pattern:
```
Ref<WebGPU::ObjectHeap> protectedObjectHeap() const { return m_objectHeap.get(); }
```

which seems essentially the same?

Reply:
(resolving as we discussed offline)